src/org/zaproxy/zap/extension/pscanrulesBeta/CookieLooselyScopedScanner.java

```diff
@@ -136,7 +136,7 @@ private void raiseAlert(HttpMessage msg, int id, String host, List<HttpCookie> l
 		}		
 		
 		alert.setDetail(getDescriptionMessage(), msg.getRequestHeader()
-				.getURI().toString(), null, getExploitMessage(), 
+				.getURI().toString(), null, "", 

```

Comment:
I suggest removing it.

1. It's really simple code.
2. It'll live on in the code repository history.